### PR TITLE
chore: compress images

### PR DESCRIPTION
**Pull request by paulatulis**
Per https://3.basecamp.com/3986751/buckets/8824800/todos/3013435881 I removed large images and replaced with compressed versions.

I did not compress gifs because it makes them look horrible. Maybe we can go through and decide what to do with each of them. There's only a handful.

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit c785651ed428cdffbe8d5e565f9abf3eea312e7b

https://deploy-preview-1611--stoplightio.netlify.app

File: netlify/blog-posts/writing-documentation-when-you-arent-a-technical-writer-—-part-one.md [hunk 1]
```diff
@@ -48,7 +48,9 @@ _In that year, I learned a lot from the Write the Docs community, other API prov
 
 ## How do people actually read documentation?
 
-![“Nation Shudders At Large Block Of Uninterrupted Text,” photo from [The Onion](https://www.theonion.com/nation-shudders-at-large-block-of-uninterrupted-text-1819571366)](/images/the-onion.png)_“Nation Shudders At Large Block Of Uninterrupted Text,” photo from [The Onion](https://www.theonion.com/nation-shudders-at-large-block-of-uninterrupted-text-1819571366)_
+![“Nation Shudders At Large Block Of Uninterrupted Text,” photo from [The Onion](https://www.theonion.com/nation-shudders-at-large-block-of-uninterrupted-text-1819571366)](/images/the-onion.jpg)_“Nation
```
Comment (texas-kristen) [inline netlify/blog-posts/writing-documentation-when-you-arent-a-technical-writer-—-part-one.md:51]: The other image on this page needs to be changed to heat-maps.jpg

`Heatmaps from [Nielsen Norman Group](/images/heat-maps.jpg)_Heatmaps from [Nielsen Norman Group](https://www.nngroup.com/articles/f-shaped-pattern-reading-web-content-discovered/)_`

**Review by texas-kristen — CHANGES_REQUESTED** [review]
See comment regarding heatmaps image on [https://deploy-preview-1611--stoplightio.netlify.app/blog/writing-documentation-when-you-arent-a-technical-writer-part-one-ef08a09870d1/](https://deploy-preview-1611--stoplightio.netlify.app/blog/writing-documentation-when-you-arent-a-technical-writer-part-one-ef08a09870d1/)

File: netlify/blog-posts/writing-documentation-when-you-arent-a-technical-writer-—-part-one.md (hunk 1 above)
Comment (paulatulis) [reply netlify/blog-posts/writing-documentation-when-you-arent-a-technical-writer-—-part-one.md:59]: Great eye thank you so much!

**Comment by paulatulis** [comment]
> @paulatulis See comment regarding heatmaps image on https://deploy-preview-1611--stoplightio.netlify.app/blog/writing-documentation-when-you-arent-a-technical-writer-part-one-ef08a09870d1/

Thanks so much for flagging. Should be updated here:
https://github.com/stoplightio/stoplight.io/pull/1611/commits/27a107dff1c15f88b07537c76214c71568649266#diff-69391cd3e2604e996c9d64573a2eec7cR80

**Review by texas-kristen — APPROVED** [review]

**Pull request merged**